flowetl/flowetl/flowetl/operators/staging/create_and_fill_staging_table.py

```diff
@@ -0,0 +1,52 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+from typing import List
+from airflow.operators.postgres_operator import PostgresOperator
+from flowetl.operators.staging.event_columns import event_column_mappings
+
+
+class CreateAndFillStagingTable(PostgresOperator):
+    """
+    Creates a new table containing only identifiers (MSISDN, IMEI, IMSI, TAC), cell_id, time of contact
+    and contact event type (defined in sql/event_type_enums.csv) from foreign tables mounted by operators
+    produced by mount_event_operator_factory.
+
+    Parameters
+    ----------
+    event_type_list:List[str]
+        A list of event types; any or all of 'call','sms','location','mds','topup'
+    """
+
+    def __init__(self, *args, event_type_list: List[str], **kwargs):
```

Comment:
What's the `*args` about?

Reply:
Given this is basically a Postgres operator with a special case, I wasn't sure how Airflow calls or passes it around internally, or if there's a convention I've missed - hence *args and **kwargs. Now there's unit tests in place, I can take them out and see if they still work.